Lib/test/test_monitoring.py

```diff
@@ -1378,33 +1378,35 @@ def func():
                     x = 4
                 else:
                     x = 6
+            7
 
         self.check_events(func, recorders = JUMP_AND_BRANCH_RECORDERS, expected = [
             ('branch', 'func', 2, 2),
-            ('branch', 'func', 3, 4),
+            ('branch', 'func', 3, 6),
```

Comment:
This is now correct. See https://github.com/python/cpython/pull/109038#issuecomment-1784945555